.github/workflows/npm.yml

```
@@ -0,0 +1,25 @@
+            - uses: DeLaGuardo/setup-clojure@12.5
+            - uses: actions/setup-node@v3
+              with:
+                node-version: "20"
+            - name: npm release
+              run: |
+                npm ci && \
+                npm run lib
+            - uses: JS-DevTools/npm-publish@v3.1.1
+              with:
+                token: ${{ secrets.NPM_TOKEN }}
+          # TODO
```

yamllint

[warning] comment not indented like content

(comments-indentation)